stable/kafdrop/values.yaml

```diff
@@ -5,7 +5,7 @@ image:
   # -- Kafdrop image repository.
   repository: obsidiandynamics/kafdrop
   # -- Kafdrop image version.
-  tag: 4.0.1
+  tag: 4.0.2-SNAPSHOT
```

Comment:
Don't change this to snapshot, change the local Helm values.

Reply:
Understood, I'll close this PR and action in the local values as suggested.